ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -103,7 +103,7 @@
 OVS_FP_BRIDGE_TEMPLATE = 'br-%s-%d'
 OVS_FP_TAP_TEMPLATE = '%s-t%d'
 OVS_BP_TAP_TEMPLATE = '%s-back'
-INJECTED_INTERFACES_TEMPLATE = 'inje-%s-%d'
+INJECTED_INTERFACES_TEMPLATE = 'inj-%s-%d'
```

Comment:
Why do you need this change? Wouldn't you again hit the limit if you use a larger size name for your testbed?

Reply:
Before we attempt to create the interface, we have a check for `VM_SET_NAME_MAX_LEN`:
```
if len(vm_set_name) > VM_SET_NAME_MAX_LEN:
                raise Exception("vm_set_name can't be longer than %d characters: %s (%d)" % (VM_SET_NAME_MAX_LEN, vm_set_name, len(vm_set_name)))
```
So if the defined name is greater than 8 characters, the existing logic would handle it and throw an exception.

I had earlier planned to change the limit `VM_SET_NAME_MAX_LEN` to 7 instead of 8. But that would take away one character from the user defined name. So instead I think taking away one char from hard-coded template will be better?